lib/components/find-a-domain-component.js

```diff
@@ -10,64 +10,80 @@ export default class FindADomainComponent extends BaseContainer {
 		super( driver, By.css( '.register-domain-step' ) );
 		this.declineGoogleAppsLinkSelector = By.className( 'google-apps-dialog__cancel-link' );
 	}
+
 	waitForResults() {
 		const driver = this.driver;
 		const resultsLoadingSelector = By.css( '.domain-suggestion.is-placeholder' );
 		driver.wait( function() {
 			return driverHelper.isElementPresent( driver, resultsLoadingSelector ).then( function( present ) {
-				return ! present;
+				return !present;
 			} );
 		}, this.explicitWaitMS * 2, 'The domain results loading element was still present when it should have disappeared by now.' );
 		return this.checkForUnknownABTestKeys();
 	}
+
 	waitForGoogleApps() {
 		return this.driver.wait( until.elementLocated( this.declineGoogleAppsLinkSelector ), this.explicitWaitMS, 'Could not locate the link to decline google apps.' );
 	}
+
 	searchForBlogNameAndWaitForResults( blogName ) {
 		const searchInputSelector = By.className( 'search__input' );
 		driverHelper.setWhenSettable( this.driver, searchInputSelector, blogName );
 		return this.waitForResults();
 	}
+
 	checkAndRetryForFreeBlogAddresses( expectedBlogAddresses, blogName ) {
 		const self = this;
 		self.freeBlogAddress().then( ( actualAddress ) => {
-			if ( expectedBlogAddresses.indexOf( actualAddress ) < 0 ) {
+			if (expectedBlogAddresses.indexOf( actualAddress ) < 0) {
```

Comment:
These extra spaces are actually required here to pass `eslint`, you can install an eslint plugin for your IDE which will give you real time eslint feedback, like I use WebStorm:

<img width="586" alt="screen shot 2018-01-25 at 4 13 37 pm" src="https://user-images.githubusercontent.com/128826/35373557-b86bff60-01ea-11e8-9d91-41e2cc7a8967.png">

Reply:
Thanks for this! I also use WebStorm and do have `eslint` plugin installed. The highlight color is too pale in my environment which makes me miss some notifications. I'll look into it. Fixed the spacing here.